qiskit_metal/draw/utility.py

```diff
@@ -37,6 +37,9 @@
     'vec_unit_planar', 'Vector'
 ]
 
+# Used for numpy.round()
+PRECISION = 10
```

Comment:
Did you mean 10 or 11?

Reply:
In another file `qiskit_metal/toolbox_metal/math_and_overrides.py`, I found the decimal precision to be 10, so just thought of making it constant across all the vector math classes. Do you think I should increase it to 11?